boards/arm/mps2_an521/mps2_an521_ns.yaml

```diff
@@ -16,3 +16,4 @@ testing:
     - kernel
     - tfm
```

Comment:
I think this can be removed, also may make sense to modify `boards/arm/mps3_an547/mps3_an547_ns.yaml` as well

Reply:
I agree, but that seems to be something which should be handled in another PR. it seems @d3zd3z PR missed those in the cleanup of tfm

Reply:
for tfm removal? sure, more worried about the other board failing somewhere else

Reply:
There are actually problems with QEMU and TF-M with the AN547 target, where there are basically two different versions of the FVP image this target attempts to replicate, and TF-M assumes different implementation choices than what QEMU models. It shouldn't be used for recent versions of TF-M, so please leave it out of these changes for now.